### ISSUE TITLE
Setting conversationAccentColor hides links

### ISSUE THREAD

**Issue by pietbrauer** [smooch/smooch-ios#89]
When users send me links they have the same color as the background of the cell. I reported this already on Twitter but I guess its better to do it here.

I set:

```swift
settings.conversationAccentColor = UIColor.gtg_primaryAccentColor() // this is purple
settings.conversationStatusBarStyle = .LightContent
Smooch.initWithSettings(settings)
```

![img_0874](https://cloud.githubusercontent.com/assets/759730/13637327/beaf7256-e641-11e5-9bb5-0a2c6f2f3893.PNG)

**Comment by mspensieri** [comment]
Looks like an oversight on our part - we never optimized for the _user_ sending links. Will fix

**Comment by pietbrauer** [comment]
No worries, it's rare but if so they always point it out to me 😀. So I thought I would open an issue.

**Comment by mspensieri** [comment]
Fixed in v4.1.2
